_positions/product-lead.md

```diff
@@ -8,6 +8,9 @@ active: true
 ###Role Summary
 An 18F Product Lead is responsible for managing the development, delivery, and continuous improvement of products while establishing and managing our relationship with our partners (clients). As a product lead, you’ll work with the 18F intake team to evaluate potential work and communicate to potential clients, partner with the finance and contracts team to close deals, and work with cross-functional teams to deliver the right product in the right way.
 
+At this time, the Product team is only looking for new members who are able to join the team at an office - Chicago, NYC, DC, San Francisco. 
```

Comment:
* Ok - will change hyphen to an em-dash 
* Ok
* checking on this now
* Ok sounds good